java-frontend/src/main/java/org/sonar/java/bytecode/se/BytecodeEGWalker.java

```diff
@@ -329,6 +329,33 @@ void executeInstruction(BytecodeCFGBuilder.Instruction instruction) {
         sv = constraintManager.createSymbolicValue(instruction);
         programState = programState.put(index, sv).addConstraint(sv, ObjectConstraint.NOT_NULL);
         break;
+      case I2L:
+      case I2F:
+      case I2D:
+      case L2I:
+      case L2F:
+      case L2D:
+      case F2I:
+      case F2L:
+      case F2D:
+      case D2I:
+      case D2L:
+      case D2F:
+      case I2B:
+      case I2C:
+      case I2S:
+        // do nothing
```

Comment:
Those instruction may have an impact on stack/values regarding : https://jira.sonarsource.com/browse/SONARJAVA-2508 
Maybe something should be added regarding those in that ticket ?

Reply:
yes, added TODO here, and updated ticket